proposals/stdlib/char-int-conversions.md

```diff
@@ -136,10 +146,15 @@ to the corresponding `Char` representation.
  * If this value is less than `10`, the decimal digit Char with code `'0'.code + this` is returned.
  * Otherwise, the uppercase Latin letter with code `'A'.code + this - 10` is returned.
  */
-fun Int.digitToChar(radix: Int = 10): Char
+fun Int.digitToChar(radix: Int): Char
 ```
 
-`isDigit` is considered to be `true` for a `Char` if the Unicode general category of the `Char` is "Nd" (`CharCategory.DECIMAL_DIGIT_NUMBER`).
+We will also introduce an overload with no arguments having implicit `radix = 10`:
+- `fun Char.digitToInt(): Int`
+- `fun Char.digitToIntOrNull(): Int?`
+- `fun Int.digitToChar(): Char`
+
+Having overloads and not a default argument will be consistent with `String.toInt/toIntOrNull()` functions.
```

Comment:
Please take this paragraph from the issue KT-23451

Reply:
done